lib/charms/mysql/v0/mysql.py

```diff
@@ -494,7 +494,9 @@ def _on_set_password(self, event: ActionEvent) -> None:
 
     def _get_cluster_status(self, event: ActionEvent) -> None:
         """Action used  to retrieve the cluster status."""
-        if status := self._mysql.get_cluster_status():
+        extended = event.params.get("extended") or 0
+
```

Comment:
value need validation, e.g.
```suggestion
if not 0 <= extended < 4:
  event.fail("Extended parameter outside valid range")
  return

```